index.js

```diff
@@ -1,41 +1,48 @@
 'use strict';
 
 const packagePath = 'node_modules/serverless-offline-direct-lambda';
-const handlerPath = `proxy.js`;
+const handlerPath = 'proxy.js';
 
 class ServerlessPlugin {
   constructor(serverless, options) {
     this.serverless = serverless;
     this.options = options;
 
     this.hooks = {
-      "before:offline:start:init": this.startHandler.bind(this),
+      'before:offline:start': this.startHandler.bind(this),
```

Comment:
This seems to work in that it allows me to just run `sls offline` instead of `sls offline start`, but `sls offline start` no longer creates the proxies. Is there any way to support both?

Reply:
I see. I just tested with both hooks there and it seems like just one of them will trigger when `sls offline` is invoked with or without `start`.